compute_tools/src/config.rs

```diff
@@ -140,13 +140,42 @@ pub fn write_postgres_conf(
         writeln!(file, "# Managed by compute_ctl: end")?;
     }
 
-    // If audit logging is enabled, configure pgaudit.
+    // If base audit logging is enabled, configure it.
+    // In this setup, the audit log will be written to the standard postgresql log.
+    if let ComputeAudit::Log = spec.audit_log_level {
```

Comment:
Should it be a `match` instead? It's just a bit confusing to see two separate if's, which are actually mutually exclusive